ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -150,3 +153,22 @@ conformance_test(
         "--include=ConfigManagementServiceIT",
     ],
 )
+
+container_image(
+    name = "app-image-base",
+    base = "@openjdk_base//image",
+    cmd = None,
+    directory = "/usr/bin",
+    files = [
+        "@com_github_grpc_ecosystem_grpc_health_probe_binary//file",
+    ],
+) if not is_windows else None
+
+java_image(
+    name = "app-image",
+    base = ":app-image-base",
```

Comment:
You can probably just use `"@openjdk_base//image"` here, and delete `app-image-base`, if you don't care about running this inside an orchestrator/scheduler such as Kubernetes.

Reply:
Done